docs/mine/lotus/dagstore.md

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Lotus Miner: About the markets dagstore'
+description: ''
+breadcrumb: 'About the markets dagstore'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+[[TOC]]
+
```

Comment:
::: tip
This guide is WIP, please create an issue if you have any feedback!
:::